packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -20,17 +19,17 @@ type SubscriptionItemProps = {
   customerSubscription: CustomerSubscription;
   subscription: Subscription | null;
   plan: Plan | null;
-  cancelSubscription: Function;
-  reactivateSubscription: Function;
-  customer: CustomerFetchState;
-  updatePaymentStatus: UpdatePaymentFetchState;
-  resetUpdatePayment: Function;
-  updatePayment: Function;
-  cancelSubscriptionMounted: Function;
-  cancelSubscriptionEngaged: Function;
-  cancelSubscriptionStatus: CancelSubscriptionFetchState;
-  updatePaymentMounted: Function;
-  updatePaymentEngaged: Function;
+  cancelSubscription: SubscriptionsProps['cancelSubscription'];
```

Comment:
After figuring out how references to properties within types work, I realized we can re-use parent component prop definitions to specify props in child components.